### ISSUE TITLE
is the source code is full mean readyy to use

### ISSUE THREAD

**Issue by Ayububi** [codingwithmuhib/MernStack-Tour-Management#1]

**Comment by This-is-Lakshya** [comment]
I don't know which code out of my repo you are talking about but, yeah all of them are ready to be used. More info about the particular one will help me to answer more effectively.

**Comment by shenaltissera** [comment]
No because the project is implemented using react instead of angular

**Comment by iamkaniska** [comment]
yes !! i want to say there are some minor issues but still this is good

**Comment by Sunildharajiya** [comment]
Please close this issue because it was not issue #1
